src/zc/relation/queryfactory.py

```diff
@@ -11,20 +11,21 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-import persistent
 import BTrees
+import persistent
+import zc.relation.catalog
+import zc.relation.interfaces
 import zope.interface
 
-import zc.relation.interfaces
-import zc.relation.catalog
```

Comment:
PEP-8 recommends three import groups separated with blank lines:

1. standard library
2. 3rd-party packages
3. the current package (zc.relation)

Reply:
They got resorted because of Plone isort config, I will revert this part